orttraining/orttraining/test/graph/gradient_graph_builder_test.cc

```diff
@@ -680,26 +714,11 @@ class PipelineSplitter {
                         name,
                         input_name)) {
         ASSERT_TRUE(is_first);
-        // add shape info
-        EXPECT_TRUE(AddItemByName(sub_gp->mutable_value_info(),
-                                  main_gp.input(),
-                                  name,
-                                  name));
```

Comment:
why remove this? it makes Netron view better...

Reply:
i can add them back. on the other hand, in general, we can't rely on value_info being present either.

---
In reply to: [406528154](https://github.com/microsoft/onnxruntime/pull/3446#discussion_r406528154) [](ancestors = 406528154)